__tests__/otp.test.ts

```diff
@@ -0,0 +1,46 @@
+import { Model } from "mongoose";
+import { IOtpDoc, IOtpModel } from "../src/interfaces/otp";
+
+const OtpModel: IOtpModel = {
+    create: jest.fn(),
+    findOne: jest.fn(),
+} as any;
+
+beforeEach(() => {
+    jest.clearAllMocks();
+});
+
+describe('Otp Model Tests', () => {
+    it('should create an OTP correctly', async () => {
+        const otpData = {
+            email: "test@example.com",
+            otp: "123456",
+            createdAt: new Date(),
+        };
+
+        (OtpModel.create as jest.Mock).mockResolvedValueOnce(otpData);
```

Comment:
If the function `create` is defined correctly at the previous declaration step, not need to convert the data type into jest.Mock again here